DEVELOPMENT.md

```diff
@@ -0,0 +1,62 @@
+## Table of Contents
+* [Development](#development)
+    * [Setup](#setup)
+    * [Reference](#reference)
+
+## Development
```

Comment:
here as well my suggestion is to make this `#` and remove the title `Development` from the table of contents
wdyt?